apps/server/src/modules/learnroom/index.ts

```diff
@@ -1,2 +1,8 @@
 export * from './learnroom.module';
-export { CommonCartridgeExportService, CourseCopyService, CourseService, RoomsService } from './service';
+export {
+	CommonCartridgeExportService,
```

Comment:
@Metauriel should all this services allowed to use external modules with all this methods?

Reply:
waiting for answer

Reply:
He say yes. But in future maybe services must be split. The seperate methods that can be use in other modules and methods that should not.